install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -82,18 +82,17 @@ UPDATE `PREFIX_product` SET `product_type` = "virtual" WHERE `is_virtual` = 1;
 before Doctrine schema update */
 /* consequently we create the table manually */
 CREATE TABLE IF NOT EXISTS `PREFIX_feature_flag` (
-  `id_feature_flag` INT(10) unsigned NOT NULL AUTO_INCREMENT,
-  `name` VARCHAR(191) COLLATE utf8mb4_general_ci NOT NULL,
-  `state` TINYINT(1) NOT NULL DEFAULT '0',
-  `label_wording` VARCHAR(191) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
-  `label_domain` VARCHAR(255) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
-  `description_wording` VARCHAR(191) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
-  `description_domain` VARCHAR(255) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
-  PRIMARY KEY (`id_feature_flag`),
-  UNIQUE KEY `UNIQ_91700F175E237E06` (`name`)
+    `id_feature_flag` INT(10) unsigned NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) COLLATE utf8mb4_general_ci NOT NULL,
+    `state` TINYINT(1) NOT NULL DEFAULT '0',
+    `label_wording` VARCHAR(255) COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
```

Comment:
@matks Why did you define it to `191` characters ?

Reply:
Sorry, I overlooked that the `name` is also at `varchar(191)`...